django/db/migrations/state.py

```diff
@@ -685,11 +685,8 @@ def name_lower(self):
         return self.name.lower()
 
     def get_field(self, field_name):
-        field_name = (
-            self.options['order_with_respect_to']
-            if field_name == '_order'
-            else field_name
-        )
+        if field_name == '_order' and 'order_with_respect_to' in self.options:
+            field_name = self.options['order_with_respect_to']
```

Comment:
```suggestion
        if field_name == '_order':
            field_name = self.options.get('order_with_respect_to', field_name)
```

Reply:
@charettes thank for your suggestion! Seems @felixxm already applied it.